pages/box_buster.vue

```diff
@@ -174,3 +192,8 @@ export default {
   }
 }
 </script>
+<style>
```

Comment:
I don't think you need your own class: https://getbootstrap.com/docs/4.0/utilities/colors/

Reply:
yes thanks will add that